### DISCUSSION TITLE
Timestamp Overlay on all mode for Gopro10 Black

### DISCUSSION THREAD

**Discussion by Siriuslit** [gopro/labs#639] [Q&A]
Hello everyone,

I successfully activated the "Add an Overlay to Video" feature on my GoPro 10 Black. However, the overlay is currently only available in video mode. I would like the overlay to be displayed in photo and time-lapse modes, just as it is in video mode.

Here is the code I have:
g1!MLFIN=0!MBRNO=10!MBURN="(0,0)[BRHH:MM:SS\ndd.mm.yyyy\n'GPS50%1.6f', 'GPS51%1.6f']"

How can I resolve this issue?

**Comment by dnewman-gpsw** [comment]
It is only available in video, and timelapse video modes.  Photos use a different processing path, and the overlays don't work there.
